graph/generate_mesh.go

```diff
@@ -188,6 +168,13 @@ func (s Service) mapEdges(fn func(int) string) []string {
 	return all
 }
 
+func (s Service) ToReachableServices(namespace string) string {
+	if len(s.Edges) == 0 {
+		return "none"
```

Comment:
nit: I'd rather use `non-existing-service` instead of `none` to not make an impression that the `none` value is somehow supported by Kuma